### DISCUSSION TITLE
Firebase Auth fails in IOS

### DISCUSSION THREAD

**Discussion by roeecohen** [capawesome-team/capacitor-firebase#782] [Q&A] [answered]
Hi,

Im trying to run my app using the capacitor firebase authntication on Xcode, but the library creates duplicated symbols error so build fails on Xcode. 

I have tried to remove the library and the build is ok. 
Please help, thanks.

**Comment by robingenz** [comment]
Thank you for creating the discussion. Please provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can debug the issue.

**Reply by roeecohen** [reply]
You can check this:
https://github.com/roeecohen/capacitor-firebase-auth-ios-fail.git

Thanks

**Reply by roeecohen** [reply]
Im using in the pod file: use_frameworks! :linkage => :static 
because dynamically provides other error

**Reply by robingenz** [reply, marked as answer]
You have not followed the installation steps correctly. I have noticed several points:

1. `use_frameworks! :linkage => :static` should not be needed
2. you don't need to call `FirebaseApp.configure()` yourself, the plugin does that
3. you do not need to install the Firebase iOS SDK yourself, the plugin does that

**Reply by roeecohen** [reply]
Thanks for your answer!
The error is gone but I have this warning:
"The default Firebase app has not yet been configured. Add `FirebaseApp.configure()` to your application initialization. This can be done in in the App Delegate's"

Is that ok?

**Reply by robingenz** [reply]
Yes, just ignore this warning.

**Reply by roeecohen** [reply]
Thanks for your help!
